### ISSUE TITLE
Laravel 8 breaks pest 

### ISSUE THREAD

**Issue by rockwellll** [nunomaduro/collision#153]
So i upgraded my Laravel 7 to 8, everything went smooth, but when i try to run `php artisan test` i get the following error 

```
   Error 

  Call to undefined method Dotenv\Repository\RepositoryBuilder::create()

```

and the path of the error is 
```
\vendor\nunomaduro\collision\src\Adapters\Laravel\Commands\TestCommand.php:135

```

**Comment by owenvoke** [comment]
What version of Collision are you using? 🤔 Sounds like it's v4 which doesn't support DotEnv 5. I think this was resolved in v5.0.0-BETA3. You can check with `composer info nunomaduro/collision`

**Comment by rockwellll** [comment]
it is version `v5.0.0-BETA2`

**Comment by owenvoke** [comment]
You should be able to run a `composer update` to update Collision to the stable v5.0.0 :+1:

If that doesn't work, I guess a `composer why-not nunomaduro/collision 5.0.0` will say why. 🤷🏻

**Comment by nunomaduro** [comment]
@A7madXatab Please re-open the issue if upgrading to 5.0.0 fixes the problem.

**Comment by rockwellll** [comment]
@nunomaduro  @owenvoke  upgrading to stable version  of collision `5.0.0` successfully solved the problem, not quite sure what happened there, must've been cache related issue, although i cleared the cache before. 

Thanks

**Comment by rockwellll** [comment]
note that also collision version `5.0.0` dosent work with laravel 7.x as it requires laravel 8.x, so i had to use collision version `5.0.0-BETA-2` to fix this issue, thought you might want to be aware of this. [check this out](https://stackoverflow.com/a/63203406/12860784)

**Comment by adedejiibrahim7** [comment]
> What version of Collision are you using? 🤔 Sounds like it's v4 which doesn't support DotEnv 5. I think this was resolved in v5.0.0-BETA3. You can check with `composer info nunomaduro/collision`

This was actually the problem. I updated collision to v5 and the problem is solved
